integration/kubernetes/helm-chart/alluxio/values.yaml

```diff
@@ -21,9 +21,9 @@ imageTag: 2.9.0-SNAPSHOT
 imagePullPolicy: IfNotPresent
 
 # Security Context
-user: 1000
-group: 1000
-fsGroup: 1000
+user: 0
+group: 0
+fsGroup: 0
```

Comment:
If the pods need to run as root to start, is there any way to configure what UID/GID are used when running `entrypoint.sh`?
- Basically I think we should make the values look like this to distinguish things:
```
securityContext:
  # Alluxio pods must run as root in order to create users & assign file ownership
  # See "alluxioUserGroup" for how to change what uid/gid are used for the Alluxio processes
  # - fsGroup is used to ensure volume mounts have the correct ownership 
  fsGroup: 1000

alluxioUserGroup:
  uid: 1000
  gid: 1000
```

Reply:
By the way forcing the Pod/Container to use the root user from https://github.com/Alluxio/alluxio/pull/15898 is slightly concerning from a security point-of-view (to put it lightly) 😅 

Can we refactor the Dockerfile to create a different admin user with permission to create users/groups & assign file ownership (i.e: a `sudo`-er), but isn't `root (uid=0, gid=0)`? The rest will still be the same; after creating the `alluxio` user & group you run the process as that user. This is just so that the container image itself doesn't run with a `root` user (which is very dangerous if people can enter the container runtime somehow).

Reply:
Added @beinan as a reviewer since he was a reviewer for https://github.com/Alluxio/alluxio/pull/15898.
Any thoughts on my comment above?